Pinta.Core/Managers/PaletteManager.cs

```diff
@@ -69,26 +69,34 @@ public Color SecondaryColor {
 		set => SetColor (false, value, true);
 	}
 
-	public Palette CurrentPalette { get; } = Palette.GetDefault ();
+	public Palette CurrentPalette { get; }
 
-	private readonly SettingsManager settings_manager;
-	private readonly PaletteFormatManager palette_format_manager;
+	private readonly ChromeManager chrome;
+	private readonly SettingsManager settings;
+	private readonly PaletteFormatManager palette_formats;
 	public PaletteManager (
-		SettingsManager settingsManager,
-		PaletteFormatManager paletteFormatManager)
+		ChromeManager chrome,
+		SettingsManager settings,
+		PaletteFormatManager paletteFormats)
 	{
 		List<Color> recentlyUsed = new (MAX_RECENT_COLORS);
 
 		recently_used = recentlyUsed;
 		RecentlyUsedColors = new ReadOnlyCollection<Color> (recentlyUsed);
 
-		settings_manager = settingsManager;
-		palette_format_manager = paletteFormatManager;
+		this.chrome = chrome;
+		this.settings = settings;
+		this.palette_formats = paletteFormats;
+
+		CurrentPalette = Palette.GetDefault ();
+
+		// This depends on `chrome` and `palette_formats`, and `CurrentPalette` having a value
```

Comment:
`this.chrome` is also unused now so we could remove that (along with updating this comment)